content/rc/changelog/december-2023.md

```diff
@@ -0,0 +1,24 @@
+---
+Title: Redis Cloud changelog (December 2023)
+linktitle: December 2023
+description: New features, enhancements, and other changes added to Redis Cloud during December 2023.
+highlights: Active-Active JSON support, Full mTLS support
+weight: 72
+alwaysopen: false
+categories: ["RC"]
+aliases: []
+---
+
+## New features
+
+### Active-Active JSON support
+
+[Active-Active databases]({{<relref "rc/databases/active-active-redis">}}) on Redis Cloud now support the [JSON]({{< relref "/stack/json" >}}) data type. 
+
+See [Create an Active-Active subscription]({{<relref "rc/subscriptions/create-active-active-subscription">}}) to learn how to create an Active-Active subscription.
+
+### Full mutual TLS (mTLS) support
+
+Databases that support [Transport layer security (TLS)]({{<relref "/rc/security/database-security/tls-ssl">}}) now support full mutual TLS (mTLS) features. When you enable mutual TLS, you can now provide certificate chains that include the root certificate and an intermediate CA without the leaf (end-entity) certificate. 
```

Comment:
I dont like the term "full mTLS" as much (I know it was used internally for this feature since so far our mTLS was somewhat hacky), but it is not an "official" term and it is not entirely clear what that means exactly. What about:

> Databases that support TLS now support multiple client certificates for use with mutual TLS. This should make it easier to rotate client certificates outside of a maintenance window. In addition, you can now provide a client certificate Certificate Authority chain without the leaf (end-entity) certificate for more flexibility.